docs/tutorials/integrations/nginx/tls/tls.conf

```diff
@@ -0,0 +1,39 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+req_extensions = req_ext
+distinguished_name = dn
+x509_extensions = v3_ca # The extentions to add to the self signed cert
+req_extensions  = v3_req
+x509_extensions = usr_cert
+
+[ dn ]
+C=US
+ST=Wisconsin
+L=Madison
+O=Onyx
+OU=CyberArk
+CN=proxy
+
+[ usr_cert ]
+basicConstraints=CA:FALSE
+nsCertType                      = client, server, email
```

Comment:
I don't think email is necessary here. Neither is client, unless we're using that to authenticate (eg. to pg).

Reply:
I tried to remove this & ran into certificate errors. See comment below.